oauth2_http/javatests/com/google/auth/oauth2/MockTokenServerTransport.java

```diff
@@ -16,10 +17,12 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import java.lang.UnsupportedOperationException;
+
 /**
  * Mock transport to simulate providing Google OAuth2 access tokens
  */
-public class MockTokenServerTransport extends MockHttpTransport {
+public class MockTokenServerTransport extends HttpTransport {
```

Comment:
This seems a little fragile in that an error could accidentially cause test code to hit real functionality. Is there a way of doing this that retains it as a pure mock?